prusti/tests/pass/typecheck/nested_forall.stdout

```diff
@@ -4,13 +4,17 @@
 
 #![feature(register_tool)]
 #![register_tool(prusti)]
+
+
+
+
+
+#![feature(stmt_expr_attributes)]
```

Comment:
So, why is this needed?

Reply:
See my explanation above for why this feature is necessary; I needed to change the `*.stdout` test files because the `#![feature(stmt_expr_attributes)]` line is added to the after-parse output when using `-Zcrate-attr=feature(stmt_expr_attributes)`.